### DISCUSSION TITLE
Idea: New Relationship Categories

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#3317] [Ideas]
### Idea Description

It would be nice to add a few more relationship categories. Namely

- One Shot
- Team-Up
- Limited Series

Keep up the great work, happy to be a monthly contributor.

### Idea Category

API

### Duration of Using Kavita

8 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Methaneus** [comment]
+1

My additions would be:
- Crossover
- One-Shot/Pilot (currently, Alternative Version does this job)
- Spoof
